### PR TITLE
fix: always undefined location value of createtion-schedule-popup(fix #615)

### LINKED ISSUE #615: Location value of default popup is always undefined
## Describe the bug
Location value of default popup is always undefined.
All of information was valid except **location** though I filled out all the inputs of default popup
## To Reproduce
1. Enable useCreationPopup option.
2. Fill out all the inputs of default popup
3. Check the log in the event listener function for beforeCreateSchedule
## Expected behavior
The value of location should not be undefined if you input location value.

## Desktop (please complete the following information):
 - OS: Mac OS X
 - Browser: chrome
 - Version 81.0.4044.138

### PR DESCRIPTION
**Pull request by gayeon1025**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
- AS-IS

image

image

- TO-BE

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

File: src/js/view/popup/scheduleCreationPopup.js [hunk 1]
```diff
@@ -764,7 +764,7 @@ ScheduleCreationPopup.prototype._onClickCreateSchedule = function(form) {
     this.fire('beforeCreateSchedule', {
         calendarId: form.calendarId,
         title: form.title.value,
-        location: location.value,
+        location: form.location.value,
```
Comment (dongsik-yoo) [inline src/js/view/popup/scheduleCreationPopup.js:767]: Oops! Thank you so much!
Can you please check `_onClickUpdateSchedule` function, too?

Comment (gayeon1025) [reply src/js/view/popup/scheduleCreationPopup.js:767]: I checked `_onClickUpdateSchedule` and there's typo, too.
I fixed it too :-)

**Comment by dongsik-yoo** [comment]
ok to test

**Comment by jungeun-cho** [comment]
[05/28] I confirmed that it works well. :)

**Pull request merged**